### DISCUSSION TITLE
What are the core technical indicators of Gateway Kong

### DISCUSSION THREAD

**Discussion by liukaiyuliukaiyu** [Kong/kong#12777] [Help] [answered]
What are the core technical indicators of Gateway Kong's stability？

**Comment by liukaiyuliukaiyu** [comment]
What are the core technical indicators of Gateway Kong's stability

**Comment by nowNick** [comment, marked as answer]
Hey @liukaiyuliukaiyu!
What do you mean by an _"indicator of Kong's stability"_ 🤔 ? If you'd like to verify if Kong's running correctly you can check `/status` endpoint of Admin API. 
API spec here: https://docs.konghq.com/gateway/api/status/latest/
